tips/TIP-0041/tip-0041.md

```diff
@@ -0,0 +1,575 @@
+          </tr>
+        </table>
+      </details>
+      <details>
+        <summary>Native Token Feature</summary>
+        <blockquote>A feature that carries a user-defined Native Token minted by a Foundry Output. Defined in <a href='../TIP-0038/tip-0038.md#native-token-feature'>TIP-38 (Native Token Feature)</a>.</blockquote>
+        <table>
+          <tr>
+            <td>
+              <b>Name</b>
+            </td>
+            <td>
+              <b>Type</b>
+            </td>
+            <td>
+              <b>Description</b>
+            </td>
+          </tr>
+          <tr>
+            <td>Feature Type</td>
+            <td>uint8</td>
+            <td>Set to <strong>value 5</strong> to denote a <i>Native Token Feature</i>.</td>
+          </tr>
+          <tr>
+            <td>Token ID</td>
+            <td>ByteArray[38]</td>
+            <td>Identifier of the native token. Its derivation is defined in <a href='../TIP-0044/tip-0044.md#foundry-output'>TIP-44 (Foundry Output)</a>.</td>
+          </tr>
+          <tr>
+            <td>Amount</td>
+            <td>uint256</td>
+            <td>Amount of native tokens of the given <i>Token ID</i>.</td>
+          </tr>
+        </table>
+      </details>
+    </td>
+  </tr>
+</table>
+
+## Additional Transaction Syntactic Validation Rules
+
+- It must hold true that `1` ≤ `Unlock Conditions Count` ≤ `4`.
+- `Unlock Condition Type` of an <i>Unlock Condition</i> must define one of the following types:
+  - <i>Address Unlock Condition</i>
+  - <i>Storage Deposit Return Unlock Condition</i>
+  - <i>Timelock Unlock Condition</i>
+  - <i>Expiration Unlock Condition</i>
+- <i>Unlock Conditions</i> must be sorted in ascending order based on their `Unlock Condition Type`.
+- Syntactic validation of all present unlock conditions must pass.
+- <i>Address Unlock Condition</i> must be present.
+- It must hold true that `0` ≤ `Features Count` ≤ `4`.
+- `Feature Type` of a <i>Feature</i> must define one of the following types:
+  - <i>Sender Feature</i>
+  - <i>Metadata Feature</i>
+  - <i>Tag Feature</i>
+  - <i>Native Token Feature</i>
+- <i>Features</i> must be sorted in ascending order based on their `Feature Type`.
+- Syntactic validation of all present features must pass.
+
+## Additional Transaction Semantic Validation Rules
+
+### Consumed Outputs
+
+- The unlock of the input must correspond to `Address` field in the <i>Address Unlock Condition</i> and the unlock must
+  be valid.
+- The unlock is valid if and only if all unlock conditions and features present in the output validate.
+
+### Created Outputs
+
+- All <i>Unlock Condition</i> imposed transaction validation criteria must be fulfilled.
+- All <i>Feature</i> imposed transaction validation criteria must be fulfilled.
+
+# Test Vectors
+
+The protocol parameters used in the following test vectors are the same as in
+[TIP-49 (Protocol Parameters Hash)](../TIP-0049/tip-0049.md#protocol-parameters-hash).
```

Comment:
dead link. probably the same in all of these examples

Reply:
Because the PR is not merged yet, it will work later

Reply:
oh 🤦‍♀️

Reply:
Yeah, sorry about that. I think I might have to put in temporary static github links for CI to pass before merging anyway, so I might update those when I have spare time.